de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantIncomingProjectNegotiation.java

```diff
@@ -31,6 +32,13 @@
 
   private static final Logger log = Logger.getLogger(InstantIncomingProjectNegotiation.class);
 
+  /**
+   * the one queuer used for all incoming project negotiations
+   *
+   * <p>TODO change to resource based queuer
```

Comment:
I am a bit confused by this comment. Could you explain what it means (in this context)?

Reply:
only one queuer exists during the whole session for all incoming negotiations

the TODO is a remark to change this queue, part of #156